### PR TITLE
adding console to deploy wasm

### PR DESCRIPTION
**Pull request by venugopv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,10 @@ language: C++
 This repository contains EDJX's sample serverless functions written in C++.
 Each sample also contains a README, which has a detailed explanation about
 the specifics of the Serverless Function. These sample functions can be
-deployed to EDJX network using EDJX CLI. More details on how to deploy via
+built and 'deployed to EDJX network' using EDJX CLI. More details on how to deploy via
```
Comment (dasebek) [inline README.md:13]: The introductory paragraph was copied from the Rust repo. Your update makes sense. Do the apostrophes added around 'deployed to EDJX network' have any meaning?

File: README.md [hunk 2]
```diff
@@ -10,9 +10,10 @@ language: C++
...
 EDJX CLI can be found [in the EDJX Documentation](https://docs.edjx.io/docs/latest/how_tos/cli_build_wasm_file.html).
 
+
```
Comment (dasebek) [inline README.md:16]: This extra line may be unnecessary

Comment (venugopv) [reply README.md]: removed the line.

File: README.md (hunk 1 above)
Comment (venugopv) [reply README.md]: I thought that would make sense.. apostrophes. But after thought.. felt not needed.

**Review by dasebek — APPROVED** [review]

**Review by d-vison — APPROVED** [review]
LGTM

**Review by d-vison — COMMENTED** [review]
LGTM

**Pull request merged**